voxblox_ros/src/intensity_server.cc

```diff
@@ -0,0 +1,123 @@
+#include "voxblox_ros/intensity_server.h"
+
+namespace voxblox {
+
+IntensityServer::IntensityServer(const ros::NodeHandle& nh,
+                                 const ros::NodeHandle& nh_private)
+    : TsdfServer(nh, nh_private),
+      focal_length_px_(391.5f),
```

Comment:
seems rather specific? Not sure it makes sense that there is a default value for the focal length? I think this should be a mandatory ros param and crash and burn if not provided.